lib/std/c/darwin.zig

```diff
@@ -27,6 +27,9 @@ pub extern "c" fn @"fstatat$INODE64"(dirfd: fd_t, path_name: [*:0]const u8, buf:
 pub extern "c" fn mach_absolute_time() u64;
 pub extern "c" fn mach_timebase_info(tinfo: ?*mach_timebase_info_data) void;
 
+pub extern "c" fn malloc_size(?*const c_void) usize;
+pub extern "c" fn posix_memalign(memptr: *?*c_void, alignment: usize, size: usize) c_int;
```

Comment:
should the first parameter be a `*?* align(@sizeOf(usize)) c_void`?

Reply:
I'm not too fond of the attributes soup, and that's the minimum alignment anyway.